common/v2/features/DeFiZap/components/ZapSelection.tsx

```diff
@@ -81,25 +125,76 @@ const ZapEducation = withRouter(({ history, location }) => {
   // };
 
   return (
-    <FullSizedContentPanel>
-      <ContentPanelHeading>{`${zapSelected.title} Details`}</ContentPanelHeading>
+    <FullSizeContentPanel width={'1100px'}>
```

Comment:
Not sure this is where it happens, but the tablet-width is way out of wack.

![image](https://user-images.githubusercontent.com/29407814/76568116-7677d880-646d-11ea-8d2f-010f17ed6a59.png)

Reply:
Yeah it's a problem with the ControlPanel Component when you set a width, the media query is fixed, gonna fix that